### PR TITLE
Ensure latests Jenkins is installed

### PR DESCRIPTION
**Pull request by Wynndow**
We want to be able to bump the version to the latest to remain on top of
security issues.

### CODE REVIEW

File: playbooks/roles/jenkins/tasks/jenkins.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@
             backup=no
 
 - name: Ensure Jenkins is installed.
-  package: name=jenkins state=present
+  package: name=jenkins state=latest
```
Comment (katstevens) [inline playbooks/roles/jenkins/tasks/jenkins.yml]: Can we stipulate a major version that `latest` can't go above? The Ansible docs aren't particularly helpful on this...

Comment (Wynndow) [reply playbooks/roles/jenkins/tasks/jenkins.yml]: interesting....i'll have a look

Comment (Wynndow) [reply playbooks/roles/jenkins/tasks/jenkins.yml]: It can be done!

Comment (katstevens) [reply playbooks/roles/jenkins/tasks/jenkins.yml]: Excellent 👍

**Pull request merged**